pkg/kf/commands/routes/unmap_route.go

```diff
@@ -81,6 +81,7 @@ func unmapApp(
 	appName string,
 	route v1alpha1.RouteSpecFields,
 	c apps.Client,
+	cmd *cobra.Command,
```

Comment:
nit: if we're just using this for the `cmd.OutOrStdout()`, could we just take in an `io.Writer` instead?

Reply:
good point, just updated the param to be io.Writer